lib/src/components/NxTransferList/TransferListHalf.tsx

```diff
@@ -48,6 +48,11 @@ function TransferListItem<T extends string | number = string>(props: TransferLis
       }
   );
 
+  const moveUpButtonTitle = isTopItem
+    ? null : isFilteredItem ? 'Reordering is disabled when filtered' : 'Move Up';
```

Comment:
IMO nested ternaries are only readable when split over multiple lines such that each line indicates a condition and the result when that condition is true, followed at the end of all the lines by the base case.  Like this:

```
  const moveUpButtonTitle = isTopItem ? null : 
      isFilteredItem ? 'Reordering is disabled when filtered' : 
      'Move Up';
```